ci_environment/docker/recipes/default.rb

```diff
@@ -11,15 +11,15 @@
 include_recipe 'travis_build_environment'
 
 apt_repository 'docker' do
-  uri 'https://get.docker.io/ubuntu'
-  distribution 'docker'
+  uri 'https://apt.dockerproject.org/repo'
+  distribution 'ubuntu-trusty'
```

Comment:
I'm not sure if there's a way for me to scrape this distro information directly.